Assets/UGSSamples/Plugins/PlayerProfile/SampleAuthenticator.cs

```diff
@@ -0,0 +1,45 @@
+
+using System.Text.RegularExpressions;
```

Comment:
Concerning the location of this file, I am not sure it fits in the PlayerProfile folder.
It is true that its a bit underwhelming to create a new folder for just this authentication, but its also weird to have it mix with this. Not sure what is the best approach

Reply:
Trying out a different file name, some kind of descriptive name that encompasses all the service-player-auth stuff is what I'm aiming for.